src/Microsoft.VisualStudio.Editors/PropPages/ReferencePropPage.vb

```diff
@@ -760,10 +760,14 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Me.AdviseReferencesEvents(CType(DTEProject.Object, VSLangProj.VSProject))
             Me.AdviseWebReferencesEvents()
             Me.AdviseServiceReferencesEvents()
-            Me.AdviseImportsEvents(CType(DTEProject.Object, VSLangProj.VSProject))
+            'TODO : Uncomment after GetCurrentImportsList() is implemented 
+            'Me.AdviseImportsEvents(CType(DTEProject.Object, VSLangProj.VSProject))
         End Sub
 
         Private Function GetCurrentImportsList(ByVal _Imports As VSLangProj.Imports) As String()
+            'TODO : Remove the below empty return after GetCurrentImportsList() is implemented 
```

Comment:
Make sure you do not check these TODO's in - this will regress existing projects.